mailcom/test/test_inout.py

```diff
@@ -1,9 +1,21 @@
 from mailcom.inout import list_of_files
+import pytest
 
 def test_list_of_files_found(tmp_path):
     p = tmp_path / "test.eml"
     p.write_text("test")
-    assert len(list_of_files(tmp_path)) != 0, "The list is empty"
+    assert len(list_of_files(tmp_path)) != 0
 
 def test_list_of_files_empty(tmp_path):
```

Comment:
would then add the test for the exception if the provided input directory does not exist